pkgs/tools/text/grip-search/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, boost, pkgconfig, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "grip-search";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner = "sc0ty";
+    repo = "grip";
+    rev = "v${version}";
+    sha256 = "0bkqarylgzhis6fpj48qbifcd6a26cgnq8784hgnm707rq9kb0rx";
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake ];
+
+  buildInputs = [ boost ];
+
+  patchPhase = ''
+    sed -i -e 's/"CUSTOM-BUILD"/${version}/' src/general/config.h
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Fast, indexed regexp search over large file trees";
+    homepage = "https://github.com/sc0ty/grip";
+    license = licenses.gpl3;
+    platforms = platforms.linux ++ platforms.windows;
```

Comment:
Not MacOS/Darwin?

Reply:
What can I say. I do not have MacOS so I am unable to test it there...

Reply:
You tested it on Windows?
I think we shouldn't exclude MacOS	because it is very likely to work.
We can exclude it, once somebody notices it doesn't work there.